CMakeLists.txt

```diff
@@ -448,11 +448,16 @@ message(STATUS "  Override with -DENABLE_INTERNAL_VOLK=ON/OFF")
 # Handle logging
 find_package(LOG4CPP REQUIRED)
 
+########################################################################
+# Disable complex math range checking
+########################################################################
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fcx-limited-range")
```

Comment:
How does this fare with clang or MSVC?

Reply:
Is this just a C++ flag, or does it also work with C? Guessing both ... We do have some C code in GR after all ... so maybe use `add_definitions` or `add_compile_options` instead? I'll test as-is on macOS using Clang.

Reply:
looking like the answer is "no" as to whether Clang supports this flag ... still testing ...

Reply:
OK so at least AppleClang doesn't support this flag. Guessing it's more clang-generic. Have to limit it to GCC!

Reply:
Let's say, we limit it to compilers that support it. I wouldn't trust all versions of gcc, either.

Reply:
Ditto for Clang & I'd guess MSVC too. Some older Clang will pass this test but then fail during the build (from experience). We have minimum compiler versions already set (and I assume checked for ...) ... might be time to review those settings on master & bump them as necessary to avoid this issue. I'd prefer less complicated code here if possible!